db/models.py

```diff
@@ -1 +1,16 @@
 from django.db import models
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+    
+
+class Actor(models.Model):
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
```

Comment:
The `__str__` method in the `Actor` model should return a string representation of the actor. Currently, it returns `self.name`, which does not exist. Consider returning a combination of `first_name` and `last_name`, such as `f'{self.first_name} {self.last_name}'`.